fiftyone/core/collections.py

```
@@ -3626,7 +3626,9 @@ def compute_metadata(
             progress=progress,
         )
 
-    def generate_label_schemas(self, fields=None, scan_samples=True):
+    def generate_label_schemas(
+        self, fields=None, scan_samples=True, scan_mode="auto"
```

Ruff (0.14.14)

[error] Boolean default positional argument in function definition

(FBT002)